05_deploy_bootstrap_vm.sh

```diff
@@ -117,15 +117,13 @@ sudo systemctl reload NetworkManager
 cat /tmp/dnsmasq.conf | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=dnsmasq.conf
 
 # Build and start the ironic container
-cat ironic/runironic.sh | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=runironic.sh
-cat ironic/Dockerfile | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=Dockerfile
-ssh -o "StrictHostKeyChecking=no" core@$IP sudo podman build \
-    --build-arg RHCOS_IMAGE_URL=${RHCOS_IMAGE_URL} \
-    --build-arg RHCOS_IMAGE_VERSION=${RHCOS_IMAGE_VERSION} \
-    --build-arg RHCOS_IMAGE_FILENAME_OPENSTACK=${RHCOS_IMAGE_FILENAME_OPENSTACK} \
-    -t ironic:latest .
+cat "${RHCOS_IMAGE_FILENAME_OPENSTACK}.gz" | ssh -o "StrictHostKeyChecking=no" "core@$IP" sudo dd of="${RHCOS_IMAGE_FILENAME_OPENSTACK}.gz"
+ssh -o "StrictHostKeyChecking=no" "core@$IP" sudo podman pull docker.io/vrutkovs/metalkube-ironic
 ssh -o "StrictHostKeyChecking=no" core@$IP sudo podman run \
-    -d --net host --privileged --name ironic localhost/ironic
+    -d --net host --privileged --name ironic \
+    -v /home/core/dnsmasq.conf:/etc/dnsmasq.conf \
+    -v "/home/core/${RHCOS_IMAGE_FILENAME_OPENSTACK}.gz:/var/www/html/images/${RHCOS_IMAGE_FILENAME_OPENSTACK}" \
```

Comment:
Sorry but I still don't think this is right - previously we had an uncompressed qcow2 for ironic to server, but now we have a gzipped version served via http without any gz extension?

AFAIK we should uncompress this, either via the curl --compress option, or before/after the copy to the bootstrap VM?

Unless there's something Ironic can do automagically with gzipped qcow2 files?  @derekhiggins maybe you can comment here.

Reply:
Right, so `curl --compressed` automatically decompressed it - so it was a proper qcow file, but stored in a `.qcow2.gz` file. Fixed filenames to avoid this confusion